doc/orchestrator.rst

```diff
@@ -1,60 +1,324 @@
+To support the apps functionality, we create a
+NumPy array in the python driver script to send to the a database. To
+accomplish this, we use ``Client.put_tensor()``:
+.. code-block:: python
+
+  # Create NumPy array
+  array_1 = np.array([1, 2, 3, 4])
+  # Use multi shard db SmartRedis client to place tensor standard database
+  driver_client_standard_db.put_tensor("tensor_1", array_1)
+
+Initialize a Model
+------------------
+In the next stage of the experiment, we
+launch the application script by configuring and creating
+a SmartSim ``Model``.
 
-A colocated Orchestrator is a special type of Orchestrator that is deployed on
-the same compute hosts an a ``Model`` instance defined by the user. In this
-deployment, the database is *not* connected together in a cluster and each
-shard of the database is addressed individually by the processes running
-on that compute host.
+Step 1: Configure
+'''''''''''''''''
+You can specify the run settings of a model.
+In this experiment, we invoke the Python interpreter to run
+the python script defined in section: The Application Script.
+To configure this into a ``Model``, we use the ``Experiment.create_run_settings()`` function.
+The function returns a ``RunSettings`` object.
+When initializing the ``RunSettings`` object,
+we specify the path to the application file,
+`application_script.py`, for
+``exe_args``, and the run command for ``exe``.
+.. code-block:: python
+
+  # Initialize a RunSettings object
+  model_settings = exp.create_run_settings(exe=exe_ex, exe_args="/lus/scratch/richaama/standard_orch_model.py")
+  model_settings.set_nodes(1)
+
+Step 2: Initialize
+''''''''''''''''''
+Next, create a ``Model`` instance using the ``Experiment.create_model()``.
+Pass the ``model_settings`` object as an argument
+to the ``create_model()`` function and assign to the variable ``model``:
+.. code-block:: python
+
+  # Initialize the Model
+  model = exp.create_model("model", model_settings)
+
+Step 3: Start
+'''''''''''''
+Next, launch the model instance using the ``Experiment.start()`` function.
+.. code-block:: python
+
+  # Launch the Model
+  exp.start(model, block=True, summary=True)
+
+.. note::
+    We specify `block=True` to ``exp.start()`` because our experiment
+    requires that the ``Model`` finish before the experiment continues.
+    This is because we will request tensors from the database that
+    are inputted by the Model we launched.
+
+Poll Data Using Clients
+-----------------------
+Next, check if the tensor exist in the standard database using ``Client.poll_tensor()``.
+This function queries for data in the database. The function requires the tensor name (`name`),
+how many milliseconds to wait in between queries (`poll_frequency_ms`),
+and the total number of times to query (`num_tries`):
+.. code-block:: python
+
+  # Retrieve the tensors placed by the Model
+  value_2 = driver_client_standard_db.poll_key("tensor_2", 100, 100)
```

Comment:
Consider changing num_tries to a smaller value (or variable) to clarify which parameter is which. Alternatively, consider keyword args. This might also reduce amount of wording in paragraph that defines the names of params